packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/AssetPreview.js

```diff
@@ -27,6 +27,14 @@ export const AssetPreview = forwardRef(({ mime, url, name }, ref) => {
     );
   }
 
+  if (mime.includes(AssetType.Audio)) {
+    return (
+      <audio controls src={url} ref={ref}>
+        <track label={name} default kind="captions" srcLang={lang} src="" />
```

Comment:
Do we need to define a `<track />` element here or would it be sufficient without?

Reply:
Maybe more consistent if I align it with AudioPreview.js and add the eslint-disable rule?
/* eslint-disable jsx-a11y/media-has-caption */

Reply:
Yes, I think that's a sensible solution. @HichamELBSI / @ronronscelestes  do you have an opinion here? I proposed to move the `alt` text in https://github.com/strapi/strapi/pull/12419#discussion_r803555535 to the children, because we don't really support tracks here ...

Reply:
Since this is the very first iteration to the feature, we can make a tradeoff and add an eslint disable line here + move the `alt` text (@gu-stav's comment) 👍🏽

Reply:
I'll fix this this evening. What's your procedure on 'finishing' pull requests (assuming tests and such won't fail) - should code be rebased or squashed or anything?

Reply:
Should be good to go - rebased and added latest change